src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -479,15 +479,18 @@ def get_platform_info(config_db=None):
     if hw_info_dict:
         return hw_info_dict
 
-    from .multi_asic import get_num_asics
+    from .multi_asic import get_asic_presence_list
 
     version_info = get_sonic_version_info()
 
     hw_info_dict['platform'] = get_platform()
     hw_info_dict['hwsku'] = get_hwsku()
     if version_info:
         hw_info_dict['asic_type'] = version_info.get('asic_type')
-    hw_info_dict['asic_count'] = get_num_asics()
+    try:
+        hw_info_dict['asic_count'] = len(get_asic_presence_list())
+    except:
+        hw_info_dict['asic_count'] = 'N/A'
```

Comment:
is this compatiable with single asic? previously for single asic, the get_num_asics() return 1, in this case, it returns 
"N/A" now?

Reply:
@lguohan yes this change was breaking single asic as it was returning 0. Have fixed API `get_asic_presence_list()` to work  for single asic.

Reply:
in the multi-asic case, what is in the asics_list? is that 0,1,2 or something else? @abdosi

Reply:
@lguohan : yes it 0,1,2